lecture/src/test/resources/config/application.yml

```diff
@@ -77,6 +77,7 @@ spring:
         host: localhost
     main:
         allow-bean-definition-overriding: true
+        allow-circular-references: true # TODO: we should deactivate this in the future, however currently securityConfiguration has a circular dependency to itself
```

Comment:
Do you have this on your TODO list to refactor it? Otherwise, or possibly either way, please create an issue for that, so it doesn't get forgotten.

Reply:
Same for the user-management

Reply:
I don't think it's a good idea to open an issue for that since the microservices are not on develop yet. The need for this change came from some previous changes applied from the develop branch which included this config in the artemis app as well. If it's solved until I am in the project, I will remove it from the config. If not, I can create an issue once the microservices are on develop.

Reply:
The reason I would suggest creating the issue now is because people tend do forget things 😅

Reply:
I understand, but I still don't think it makes sense creating an issue for implementation that is not in develop yet.